src/SelfService/Infrastructure/Api/Capabilities/CapabilityController.cs

```diff
@@ -23,6 +25,7 @@ public class CapabilityController : ControllerBase
     private readonly IKafkaClusterRepository _kafkaClusterRepository;
     private readonly IKafkaTopicRepository _kafkaTopicRepository;
     private readonly ITeamApplicationService _teamApplicationService;
+    private readonly IInvitationApplicationService _invitationApplicationService;
     private readonly ILogger<CapabilityController> _logger;
     private readonly IMembershipApplicationService _membershipApplicationService;
     private readonly ICapabilityMembersQuery _membersQuery;
```

Comment:
would prefer if the ordering of the fields and parameters in the constructors were sorted the same

Reply:
Sure, we can start doing this.
I have moved around the one related to this PR.
If we want to move around all our parameters for all our definitions, then lets do this as a separate cleanup-task